src/js/msp.js

```diff
@@ -341,16 +340,14 @@ const MSP = {
             'start': performance.now(),
         };
 
-        if (!requestExists) {
```

Comment:
Still testing this :)

Don't think it is a needed to remove the timer.

This PR:

![image](https://github.com/betaflight/betaflight-configurator/assets/8344830/5b7e73c9-aebc-477b-937c-090b5f9f1757)

Would be better to only add a timer when the request does not exist.

![image](https://github.com/betaflight/betaflight-configurator/assets/8344830/318b095a-d994-41ac-b940-9293e025b589)

Reply:
In this case we are risquing to get old message to be transfered on timer, after the new masses has gone.

If I only improve increase slider timeout, the issue still persist, altho way more rare.

Reply:
Agree setting silder timeout below MSP.timeout is not optimal.
We also might suppress `update_live_status` from running when armed.
Removing timeout from a request can break some things where we need a sequence of the same requests.

Reply:
> Agree setting silder timeout below MSP.timeout is not optimal. We also might suppress `update_live_status` from running when armed. Removing timeout from a request can break some things where we need a sequence of the same requests.

Yeah but it's not removing timeout completely. It replaces it with the fresher command timeout of the same type.

Reply:
> > Agree setting silder timeout below MSP.timeout is not optimal. We also might suppress `update_live_status` from running when armed. Removing timeout from a request can break some things where we need a sequence of the same requests.
> 
> Yeah but it's not removing timeout completely. It replaces it with the fresher command timeout of the same type, and newer data

Reply:
We are using `input mouseup` event for sliders. Not sure `input wheel` has a negative effect here.

Reply:
We are using `input mouseup` event for sliders in pid-tuning tab.

Not sure `input wheel` has a negative effect here.